rmf_demos/launch/common.launch.xml

```diff
@@ -15,10 +15,6 @@
   <node pkg="rmf_traffic_ros2" exec="rmf_traffic_schedule" output="both" name="rmf_traffic_schedule_primary">
     <param name="use_sim_time" value="$(var use_sim_time)"/>
   </node>
-  <!-- Traffic Schedule monitor and replacement node -->
-  <node pkg="rmf_traffic_ros2" exec="rmf_traffic_schedule_monitor" output="both" name="rmf_traffic_schedule_backup">
-    <param name="use_sim_time" value="$(var use_sim_time)"/>
-  </node>
```

Comment:
Admittedly I don't know much about how this node works but any reason why it was removed? Was it due to the liveliness callback getting triggered spuriously once in a while and initiating fallback behavior?